Dev/GitHub/notifications.30s.py

```diff
@@ -1,30 +1,71 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Github Notifications by Keith Cirkel
-# Fetch Github notifications in your menu bar!
+
+# <bitbar.title>GitHub Notifications</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
```

Comment:
What's the recommendation for a plugin with multiple authors (especially one with a change so drastic since @keithamus first wrote it)? I didn't want to assume Keith still wants to be on the hook to support a plugin that now looks much different.